### DISCUSSION TITLE
Panel not removing

### DISCUSSION THREAD

**Discussion by maheshnandaniya** [GrapesJS/grapesjs#4284] [Q&A]
`var editor = grapesjs.init({
         panels : {
           defaults : []
         },
});`
this code can't remove panels from editor in grapesjs version "0.18.4"  
please help me to remove panels from editor..

**Comment by patraprerna31** [comment]
You can refer this link: [https://grapesjs.com/docs/api/panels.html#removepanel](url)

**Reply by maheshnandaniya** [reply]
We can remove the right  panel using css
`.gjs-pn-views-container{
  display: none;
}`
It works fine in grapesjs version 0.18.4
![Screenshot_2022-05-09-15-13-07-101_com android chrome](https://user-images.githubusercontent.com/76687572/167384349-f1a54a25-6d63-4026-abf5-649c04c12423.jpg)
